modules/dsp/NE10_fir.neon.s

```diff
@@ -532,10 +541,19 @@ dInp2_1          .dn   D31.F32
                     LDR         pState,[pStateStruct],#4
 
                     @//outBlockSize = blockSize / S->M
+#ifdef __PIC__
+                    @/* position-independent access of LDR pMask,=ne10_divLookUpTable */
+                    LDR         pTemp,.L_PIC1_GOT_OFFSET
+                    LDR         pMask,.L_GOT_ne10_divLookUpTable
+.L_PIC1:
+                    ADD         pTemp,pTemp,pc
+                    LDR         pMask,[pTemp,pMask]
+#else
                     LDR         pTemp,=ne10_divLookUpTable
```

Comment:
@Brahmasmi Sorry, I cannot answer without more investigation. At least, we should write a test code first.